bcgov/moh-prime
# PRIME-2119 Add loading indicator to all submissions

[PRIME-2119](https://bcgovmoh.atlassian.net/browse/PRIME-2119)

### How to test:
Try all the different submissions we have (enrolment, site registration, health authority site registration). You should see the spinner with a message "Wait patiently, the submission could take up to a minute." only on the last, submission, page
